hash/nes.xml

```diff
@@ -61592,7 +61592,7 @@ preliminary proto for the PAL version, still running on NTSC systems) or the gfx
 
 <!-- nointro -->
 	<software name="mjcompan">
-		<description>Mahjong Companion (Jpn, Hacker)</description>
+		<description>Mahjong Companion (Japan, hacker)</description>
```

Comment:
Can we leave this "Hacker" capitalized? It's the publisher name, Hacker International.